bin/create-gutenberg-test-post.php

```diff
@@ -0,0 +1,168 @@
+<?php
+/**
+ * Creates a post to test all Gutenberg blocks.
+ *
+ * @codeCoverageIgnore
+ * @package AMP
+ */
+
+/**
+ * Gets many of the Gutenberg fixture blocks in blocks/tests/fixtures/.
+ *
+ * @throws Exception If this is script is not run inside the plugin directory.
+ * @return string $content Post content with all Gutenberg blocks.
+ */
+function amp_get_blocks() {
+	$fixtures_dir = dirname( dirname( __DIR__ ) ) . '/gutenberg/blocks/test/fixtures';
+	$content      = amp_get_block_permutations();
+	if ( ! is_dir( $fixtures_dir ) ) {
+		$fixtures_dir = dirname( $fixtures_dir );
+		if ( ! is_dir( $fixtures_dir ) ) {
+			throw new Exception( 'Please run this script from the AMP plugin root.' );
+		}
+	}
+
+	foreach ( glob( $fixtures_dir . '/*.html' ) as $file ) {
+		if ( ! preg_match( '/(serialized|embed|shortcode|custom-text-teaser|core__block)/', $file ) ) {
+			// Add the block's title.
+			preg_match( ':core__(?P<block>.+)\.html:s', basename( $file ), $matches );
+			if ( isset( $matches['block'] ) ) {
+				$content .= sprintf( '<h1>%s</h1>', $matches['block'] );
+			}
+			$content .= file_get_contents( $file ); // @codingStandardsIgnoreLine: file_get_contents_file_get_contents, file_system_read_file_get_contents.
+		}
+	}
+
+	// Replace broken URLs in fixture files.
+	$content = str_replace( 'http://google.com/hi.png', 'https://cldup.com/-3VMmmrPm9.jpg', $content );
+	$content = str_replace( 'https://awesome-fake.video/file.mp4', 'https://videos.files.wordpress.com/DK5mLrbr/video-ca6dc0ab4a_hd.mp4', $content );
+	return $content;
+}
+
+/**
+ * Gets the Gutenberg block permutations.
+ *
+ * These are mostly copied from gutenberg/blocks/test/fixtures/, and slightly modified.
+ * Embeds and shortcodes are tested in a separate script, so this does not have have many.
+ *
+ * @return string $content The blocks as HTML.
+ */
+function amp_get_block_permutations() {
+	$blocks = array(
+		array(
+			'title'   => '(Reusable) Block With Video',
+			'content' => sprintf( '<!-- wp:block {"ref":%d} /-->', amp_create_reusable_block() ),
+		),
+		array(
+			'title'   => 'Categories With Dropdown',
+			'content' => '<!-- wp:core/categories {"showPostCounts":false,"displayAsDropdown":true,"showHierarchy":false} /-->',
+		),
+		array(
+			'title'   => 'Columns, With 2 Columns',
+			'content' => '<!-- wp:columns {"columns":2} --><div class="wp-block-columns has-2-columns"><!-- wp:paragraph {"layout":"column-1"} -->	<p class="layout-column-1">Column One, Paragraph One</p><!-- /wp:paragraph --><!-- wp:paragraph {"layout":"column-1"} --><p class="layout-column-1">Column One, Paragraph Two</p><!-- /wp:paragraph --><!-- wp:paragraph {"layout":"column-2"} --><p class="layout-column-2">Column Two, Paragraph One</p><!-- /wp:paragraph --></div><!-- /wp:columns -->',
```

Comment:
This might need to be `wp:core/columns` instead of `wp:columns`, and likewise for the rest of the line.

Reply:
Applied with [3e41cca](https://github.com/Automattic/amp-wp/pull/1018/commits/3e41cca7395724274a3efb7833ce14a1d7ee70d0).

Reply:
Gutenberg creates blocks with `wp:columns`: https://github.com/Automattic/amp-wp/pull/1019/files#diff-ae6e35f6dd94607ca0bb8954533a1cb5R174

But they are synonymous. The `core/` is implict when no namespace is present: https://github.com/WordPress/gutenberg/blob/11187b4093df792f4a5caefc765acca98d5fbb04/lib/blocks.php#L163-L164